runbotics/runbotics-orchestrator-ui/src/src-app/Actions/excel.actions.ts

```diff
@@ -281,6 +281,60 @@ const getExcelActions: () => Record<string, IBpmnAction> = () => ({
             },
         },
     },
+    'excel.insertColumnsBefore': {
+        id: ExcelAction.INSERT_COLUMNS_BEFORE,
+        label: translate('Process.Details.Modeler.Actions.Excel.InsertColumnsBefore.Label'),
+        script: ExcelAction.INSERT_COLUMNS_BEFORE,
+        runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
+        form: {
+            schema: {
+                type: 'object',
+                properties: {
+                    input: {
+                        title: translate('Process.Details.Modeler.Actions.Common.Input'),
+                        type: 'object',
+                        properties: {
+                            column: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.StartColumn'),
+                                type: 'string',
+                            },
+                            amount: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.ColumnsAmount'),
+                                type: 'number',
+                            },
+                            worksheet: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.Worksheet'),
+                                type: 'string',
+                            }
+                        },
+                        required: ['column', 'amount'],
```

Comment:
HA! here amount is required. Decide whether it should be or not

Reply:
true, forgot to remove optional from the type